src/services/git/GitHubService.ts

```diff
@@ -174,9 +174,35 @@ export class GitHubService implements ICVSService {
    *
    * The response will include a verification object that describes the result of verifying the commit's signature.
    * To see the included fields in the verification object see https://octokit.github.io/rest.js/#pagination.
+   *
+   * Sha can be SHA or branch name.
    */
-  async getRepoCommits(owner: string, repo: string) {
-    return this.unwrap(this.client.repos.listCommits({ owner, repo }));
+  async getRepoCommits(owner: string, repo: string, sha?: string): Promise<Paginated<Commit>> {
+    let url = 'GET /repos/:owner/:repo/commits';
+    if (sha !== undefined) {
+      url = `${url}?state=${sha}`;
```

Comment:
use https://www.npmjs.com/package/qs#stringifying

Reply:
Fixed here https://github.com/DXHeroes/dx-scanner/pull/164/files#diff-e8b7dc86926ce489d3e57e63817c7979R79

Reply:
not yet merged `master`?